apps/src/applab/designElements/DefaultScreenButtonPropertyRow.jsx

```diff
@@ -6,6 +7,11 @@ var DefaultButtonPropertyRow = React.createClass({
   },
 
   render: function () {
+    if (elementUtils.getId(document.querySelector('#designModeViz .screen')) ===
+      this.props.screenId) {
```

Comment:
nit: style is to indent this line an extra two chars to differentiate from contents within the block